demo/Dockerfile

```diff
@@ -14,6 +14,7 @@ COPY talkingdata-adtracking-fraud-detection /work/talkingdata/
 
 ENV LANG=en_US.UTF-8
 ENV SPARK_HOME=/work/openmldb/spark-3.2.1-bin-openmldbspark
+ENV OPENMLDB_VERSION="${OPENMLDB_VERSION}"
 ARG OPENMLDB_VERSION=0.6.3
```

Comment:
two variable OPENMLDB_VERSION?

Reply:
`ARG OPENMLDB_VERSION` is only available during the docker building time, while `ENV OPENMLDB_VERSION` will be available in the docker container as environmental variable.